consensus/src/chained_bft/block_storage/block_store.rs

```diff
@@ -440,10 +420,11 @@ impl<T: Payload> BlockStore<T> {
     /// Can't be used in production, because production insertion potentially requires state sync
     pub fn insert_vote_and_qc(
```

Comment:
Do we need this function? It doesn't look clean (the mutable reference part), I'm wondering whether we could get away without it.

Reply:
this is for testing, but we probably should migrate too

Reply:
This function is removed